WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/logic/interval/IntervalManager.java

```diff
@@ -10,8 +10,8 @@
 public class IntervalManager extends IDEIntervalManagerBase {
```

Comment:
Not necessarily indeed. Following the naming discussion above, we could for instance create one WatchDogItemManager that contains methods for both Events and Intervals. 

The EC and IJ implementations of IntervalManager are indeed almost the same. However, the setEndingDocumentOf method uses different implementations of DocumentCreator and EditorWrapper, which I think cannot be generalized as they directly depend on the IJ/EC API.